rest_sensors.cpp

```diff
@@ -1466,6 +1472,12 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                             }
                         }
                     }
+                    else if (devManaged && rsub) // Managed by DDF ?
+                    {
+                        change.addTargetValue(rid.suffix, data.string);
+                        rsub->addStateChange(change);
+                        updated = true;
+                    }
```

Comment:
I fear this is duplicate code. Also, please do take note that the DDF requires some meta tag to properly work. Not too sure why this has been introduced or if this still makes sense.

Reply:
Right, was probably added before the change with meta.
Code removed and DDF updated